### DISCUSSION TITLE
Hi Guys, 6 years old requested feature, please consider it 4873. Would be nice to have it in a next release

### DISCUSSION THREAD

**Discussion by georgerdprj** [microsoft/vscode-discussions#134] [Extension Development QnA] [answered]
Hello there.
It seems that this feature request has been completely forgotten and neglected.
I am an extension developer like many other devs out there. 
There is an option which can be exposed to the workbench editor so we can leverage the readOnly feature.
I think that all we need is someone to actually look into it.

https://github.com/microsoft/vscode/issues/4873

Please do consider this for a next release. All the best.

**Comment by isidorn** [comment, marked as answer]
This is nothing related to extensions. Will have to mark this one as answered, and I suggest to continue the discussion in the vscode issue. Thanks!
